tools/topology/sof/pipe-volume-demux-playback.m4

```diff
@@ -64,7 +63,7 @@ W_PGA(1, PIPELINE_FORMAT, DAI_PERIODS, 2, playback_pga_conf, SCHEDULE_CORE,
 
 # Mux 0 has 2 sink and source periods.
 W_MUXDEMUX(0, 1, PIPELINE_FORMAT, 2, 2, SCHEDULE_CORE,
-	LIST(`         ', "DEMUX"))
+	LIST(`         ', concat(`DEMUX', PIPELINE_ID)))
```

Comment:
@xiulipan, we might need to change the demux_priv definition where using the pipe-volume-demux-playback.m4 also.

```
diff --git a/tools/topology/sof-apl-demux-pcm512x.m4 b/tools/topology/sof-apl-demux-pcm512x.m4
index 642269827..dd185a137 100644
--- a/tools/topology/sof-apl-demux-pcm512x.m4
+++ b/tools/topology/sof-apl-demux-pcm512x.m4
@@ -49,7 +49,7 @@ define(matrix2, `ROUTE_MATRIX(5,
                             `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,0 ,1)')')
 
 dnl name, num_streams, route_matrix list
-MUXDEMUX_CONFIG(demux_priv, 2, LIST(`  ', `matrix1,', `matrix2'))
+MUXDEMUX_CONFIG(demux_priv_1, 2, LIST(`        ', `matrix1,', `matrix2'))
 
 #
 # Define the pipelines
diff --git a/tools/topology/sof-cml-demux-rt5682.m4 b/tools/topology/sof-cml-demux-rt5682.m4
index 3ea8b03d4..0d9f910ba 100644
--- a/tools/topology/sof-cml-demux-rt5682.m4
+++ b/tools/topology/sof-cml-demux-rt5682.m4
@@ -49,7 +49,7 @@ define(matrix2, `ROUTE_MATRIX(5,
                             `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,0 ,1)')')
 
 dnl name, num_streams, route_matrix list
-MUXDEMUX_CONFIG(demux_priv, 2, LIST(`  ', `matrix1,', `matrix2'))
+MUXDEMUX_CONFIG(demux_priv_1, 2, LIST(`        ', `matrix1,', `matrix2'))
 
 #
 # Define the pipelines
diff --git a/tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4 b/tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4
index 4d0927025..1bbb86eff 100644
--- a/tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4
+++ b/tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4
@@ -51,7 +51,7 @@ define(matrix2, `ROUTE_MATRIX(4,
                             `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,0 ,1)')')
 
 dnl name, num_streams, route_matrix list
-MUXDEMUX_CONFIG(demux_priv, 2, LIST(`  ', `matrix1,', `matrix2'))
+MUXDEMUX_CONFIG(demux_priv_3, 2, LIST(`        ', `matrix1,', `matrix2'))
 ')
 
 #
diff --git a/tools/topology/sof-tgl-max98357a-rt5682.m4 b/tools/topology/sof-tgl-max98357a-rt5682.m4
index 9b7be92c5..2292486bc 100644
--- a/tools/topology/sof-tgl-max98357a-rt5682.m4
+++ b/tools/topology/sof-tgl-max98357a-rt5682.m4
@@ -52,7 +52,7 @@ define(matrix2, `ROUTE_MATRIX(9,
                             `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,0 ,1)')')
 
 dnl name, num_streams, route_matrix list
-MUXDEMUX_CONFIG(demux_priv, 2, LIST(`  ', `matrix1,', `matrix2'))
+MUXDEMUX_CONFIG(demux_priv_1, 2, LIST(`        ', `matrix1,', `matrix2'))
 
 #
 # Define the pipelines
```